harmonic/install_windows_src.md

```diff
@@ -21,14 +20,17 @@ You should be able to use `TPE` as a physics engine without any issues
    You will likely want to check the box to add `conda` to your `PATH`
    during the installation process so that you won't have to do this step manually.
 
-2. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+2. Install [Visual Studio 2019 or 2022](https://visualstudio.microsoft.com/downloads/).
    The Community version is free for students, open-source developers, and personal
    development. Check "Desktop development with C++" in the Workloads tab,
    check "MFC and ATL support", and uncheck "C++ Cmake Tools." We will install
-   cmake via Conda.
+   cmake via Conda. All other checkboxes can be left unchecked.
+   It is recommended to choose the Visual Studio version based on
+   the supported versions for your ROS distribution if you plan to use Gazebo with
+   ROS.
```

Comment:
I am not sure if we want to mention this. With all probability, this binaries of Gazebo are not ABI compatible with any ROS on Windows distribution except for RoboStack, and even if it is, it is probably just by luck and this could be break at any time due to ABI changes in conda-forge packages.

Reply:
Oh, I wasn't meaning any kind of ABI compatibility. This was much more meant as a newbie advice so that they don't need to install two versions of VS, one for Gazebo and one for ROS.

Reply:
I am afraid that most people will read "if you plan to use Gazebo with ROS." as using [`ros_gz`](https://github.com/gazebosim/ros_gz)

Reply:
Exactly. Is that a problem?

Reply:
Unless I am missing something, that sentence seems to be hint that in general Windows ROS distributions will work with Gazebo installed from source via conda-forge dependency, i.e. reading that I would expect that following the official Windows install instructions (i.e. https://docs.ros.org/en/iron/Installation/Windows-Install-Binary.html) I can install ros_gz (perhaps from source) and everything could work. In reality, the two distributions may install different version (with possibly different ABI) of each shared package (for example `yaml-cpp`) that can lead to runtime segfaults.

Reply:
Ahh, ROS decided to go with chocolatey... So this could really be a problem. So do you suggest completely removing any mention of ROS? Or better saying that this guide should not be used if ros_gz is to be built?

Reply:
The easiest solution is just not to mention ROS, so I would go at least for that.  The only reasonable way that I see for getting ros_gz to work with gz installed with conda-forge dependencies is either to build all of ROS2 with conda-forge dependencies, or use RoboStack (but there ros_gz is not working at the moment, see https://github.com/RoboStack/ros-humble/issues/68), so the situation is not easy to explain.

Reply:
On the other hand, if it's not mentioned, people might just try it, losing lots of time and being confused. What about changing the wording to:

> It is recommended to choose the Visual Studio version based on the supported versions for your ROS 2 distribution if you plan to use Gazebo with ROS 2. However, be aware that the default Binary ROS 2 installation guide uses vcpkg for installation of the dependencies, and that is not compatible with conda-installed packages from this guide. To have Gazebo and ROS 2 both working on the same system, you need to choose either vcpkg, or conda (or source builds) and use it for the dependencies of both Gazebo and ROS 2. Unfortunately, to this date, there is no guide for this approach available.

Reply:
I like it, thanks! 

Regarding the sentence:

> be aware that the default Binary ROS 2 installation guide uses vcpkg for installation of the dependencies, and that is not compatible with conda-installed packages from this guide. 

I would rephrase it as:

> be aware that the default Binary ROS 2 installation guide does not use conda-forge dependencies for installation of the dependencies does, and that is not compatible with conda-forge packages from this guide. 

For these reasons:
* the installation https://docs.ros.org/en/iron/Installation/Windows-Install-Binary.html does not use vcpkg for dependencies, but just a mix of manually compiled libraries, some delivered via Chocolatey and some that need to be installed manually by downloading .zip files. 
* (this is mostly a nitpick, feel free to ignore) what ensure that the packages that we install are abi-compatible is not that they are installed via the conda package manager, but rather that they are installed via the conda-forge "distribution".

Reply:
So, I agree that we shouldn't mention ROS on this page. I mean, at all. There is a page dedicated to ROS and Gazebo interoperability, and installing them together. Anything ROS should go there https://gazebosim.org/docs/harmonic/ros_installation

The Gazebo installation pages should be Gazebo-only. Otherwise there will be no end of mixing ROS instructions in the Gazebo website. That would just be very unmaintainable.

Reply:
Ok, Mabel's argument makes sense. I'll remove the ROS mentions.